Scripts/datatypes/purpose.py

```diff
@@ -121,6 +121,8 @@ def __init__(self, specification, zone_data, resultdata, is_agent_model):
         else:
             self.model = logit.ModeDestModel(
                 zone_data, self, resultdata, is_agent_model)
+            self.access_model = logit.AccessibilityModel(
```

Comment:
Consider renaming the variable as `accessibility_model` instead of `access_model`.